Engine/GameObjects/Creatures/Attributes/AttributeSet.cs

```diff
@@ -8,6 +8,7 @@ namespace Text_Based_Adventure.Engine.GameObjects.Creatures.Attributes
 {
     public enum Attribute
     {
+        None,
```

Comment:
If I ever persist enum values to a storage mechanism that doesn't understand the C# type system, I explicitly assign numeric values to each enum member so that stored data doesn't get messed up if I insert a new value in the future (the C# compiler assigns numbers to each value behind the scenes in the order they appear in the source code).

Reply:
never thought about that. its a good point :)